scopes/generator/generator/templates.cmd.ts

```diff
@@ -17,12 +15,19 @@ export class TemplatesCmd implements Command {
   alias = '';
   loader = true;
   group = 'development';
-  options = [] as CommandOptions;
+  options = [['s', 'show-all', 'show hidden templates']] as CommandOptions;
 
   constructor(private generator: GeneratorMain) {}
 
-  async report() {
-    const results = await this.generator.listComponentTemplates();
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async report(args: [], linterOptions: TemplatesOptions) {
```

Comment:
I think the argument name should be different (`linterOptions`)?

Reply:
woops, good catch thanks, fixed !